examples/rabbitmq-transaction-consumer/rabbitmq_transaction_consumer.bal

```diff
@@ -5,27 +5,27 @@ import ballerina/rabbitmq;
 @rabbitmq:ServiceConfig {
     queueConfig: {
         queueName: "MyQueue"
-    },
-    ackMode: rabbitmq:CLIENT_ACK
+    }
 }
 // Attaches the service to the listener.
-service testSimpleConsumer on new rabbitmq:ChannelListener({ host: "localhost", port: 5672 }) {
+service transactionConsumer on new rabbitmq:Listener({ host: "localhost", port: 5672 }) {
 
     // Gets triggered when a message is received by the queue.
     resource function onMessage(rabbitmq:Message message) {
         transaction {
-            var msg = message.getTextContent();
-            if (msg is string) {
-                log:printInfo("The message received: " + msg);
+            var messageContent = message.getTextContent();
+            if (messageContent is string) {
+                log:printInfo("The message received: " + messageContent);
             } else {
-                log:printError("Error occurred while retrieving the message content");
+                log:printError("Error occurred while retrieving the message content.");
             }
             // Positively acknowledges a single message.
+            // The acknowledgement gets committed upon successful execution of the transaction,
```

Comment:
```suggestion
            // The acknowledgment gets committed upon successful execution of the transaction
```

Reply:
"acknowledgement" spelled here similar to this: [rabbitmq ack](https://www.rabbitmq.com/confirms.html)